packages/ui/src/Button.tsx

```diff
@@ -14,7 +14,7 @@ import styles from './Button.module.scss'
 export type ButtonKind = 'primary' | 'secondary' | 'danger' | 'transparent'
 
 export type ButtonVariant = 'contained' | 'outlined' | 'text'
-export type ButtonColor = 'primary' | 'secondary' | 'warning' | 'brand'
+export type ButtonColor = 'primary' | 'secondary' | 'warning' | 'brand' | 'brandCustom4'
```

Comment:
We need a better naming
How about `"authPrimary"`?

Reply:
done